### PR TITLE
refactor: slight renovate adjustments

### PR DESCRIPTION
**Pull request by h1dden-da3m0n**
### Description

This PR aims to slightly simplify / cleanup the ORG's Renovate presets.

### Changes

* use predefined presets where possible
* unify the packageRules matcher

### Issues

* n/a

### CODE REVIEW

File: renovate-presets/default.json [hunk 1]
```diff
@@ -3,22 +3,21 @@
...
     "group:testNonMajor",
-    "group:monorepos"
+    "group:monorepos",
+    "helpers:pinGitHubActionDigests"
   ],
   "addLabels": ["dependencies"],
-  "dependencyDashboard": false,
   "internalChecksFilter": "strict",
-  "prHourlyLimit": 2,
   "rebaseWhen": "conflicted",
```
Comment (ferferga) [inline renovate-presets/default.json:20]: Isn't a good idea to do this whenever the PR is behind the base branch? So all CI results are always up to date with the dependency updates.

File: renovate-presets/default.json [hunk 2]
```diff
@@ -3,22 +3,21 @@
...
+    ":prHourlyLimit2",
+    ":prConcurrentLimit20",
     ":semanticCommits",
     ":semanticCommitScope(deps)",
+    ":timezone(Etc/UTC)",
     "docker:enableMajor",
     "group:testNonMajor",
-    "group:monorepos"
+    "group:monorepos",
+    "helpers:pinGitHubActionDigests"
```
Comment (ferferga) [inline renovate-presets/default.json:16]: Why do you think version numbers are not enough?

File: renovate-presets/default.json [hunk 3]
```diff
@@ -3,22 +3,21 @@
   "description": "Default preset for use with Jellyfin's repos",
   "extends": [
     "config:base",
-    "helpers:pinGitHubActionDigests",
-    ":rebaseStalePrs",
+    ":disableDependencyDashboard",
+    ":enableVulnerabilityAlertsWithLabel(security)",
+    ":prHourlyLimit2",
+    ":prConcurrentLimit20",
```
Comment (ferferga) [inline renovate-presets/default.json:9]: I would remove this. A max of 4 PRs are possible with this config, which I think is really manageable:

* major updates
* minor and patch updates
* CI minor updates
* CI major updates

The fact that Renovate is able to track updates as soon as they're published in the registry is one of the biggest advantages it has in my opinion. Limiting them to 2 per hour defeats that purpose and might make us less resilient to 0-day vulnerabilities that dependencies might have.

Anything will improve the current situation that we have with Dependabot, renovate might also create and spam quite a few PRs, but it's way different in that regard. That's why I would remove this.

File: renovate-presets/default.json (hunk 1 above)
Comment (h1dden-da3m0n) [reply renovate-presets/default.json:20]: if we where to default too `behind-base-branch` then that would be very noisy no? as not every preset groups every dependency and every rebase/force push to a update PR triggers a notification, no?
You can always overwrite it in the used config in the end, this really is just the general global preset.

File: renovate-presets/default.json (hunk 2 above)
Comment (h1dden-da3m0n) [reply renovate-presets/default.json:16]: that was in the preset b4 this and I dont feel strongly about it, but I get why you would want to use the Git SHA's over a tag that can move at any moment.

File: renovate-presets/default.json (hunk 3 above)
Comment (h1dden-da3m0n) [reply renovate-presets/default.json:9]: realistically, even if we would get a PR with a fix for an eventual 0-day the moment it gets published to a repository, would we even be able to publish a release within the next 2 hours ourself?
I don't fell strongly about it, and can remove the hourly limit and will do so shortly :)

**Review by joshuaboniface — APPROVED** [review]
Looks good to me

**Pull request merged**